py/kubeflow/testing/run_e2e_workflow.py

```diff
@@ -590,12 +642,48 @@ def main(unparsed_args=None):  # pylint: disable=too-many-locals
     default="tektoncd",
     help="Optional Tekton namespace to use")
 
+  ############ Other cloud provider like AWS/IBM/Arrikto
+  parser.add_argument(
+    "--cloud_provider",
+    type=str,
+    default="", ## can be "aws" / "ibm" / ....
+    help="Option to enable other cloud provider functionalities aws/ibm/cisco ..."
+  )
+
+  parser.add_argument(
+    "--aws_region",
+    type=str,
+    default="us-west-2",
+    help="region containing the EKS cluster to use to run the workflow."
+  )
+
+  parser.add_argument(
```

Comment:
Can we get ride of these arguments? I think this makes `run_e2e_workflow.py` kind of heavy.

Alternatively, user can add envs to their steps to control the number of the nodes.

Reply:
Yeah it makes sense, will fix in new commit